index.js

```diff
@@ -37,14 +39,26 @@ module.exports = function(opts) {
 				// Readiness if the broker started successfully.
 				// https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-readiness-probes
 				res.writeHead(state == "up" ? 200 : 503, resHeader);
+				res.end(JSON.stringify(content, null, 2));
 			} else {
 				// Liveness if the broker is not stopped.
 				// https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-a-liveness-command
-				res.writeHead(state != "down" ? 200 : 503, resHeader);
+				if (typeof opts.liveness.checker === 'function') {
```

Comment:
This check should be done when setting the `liveness.checker` option during middleware creation, there is no point accepting a `liveness.checker` that is not a function during middleware creation.